client/src/components/FieldHelper.js

```diff
@@ -50,7 +51,7 @@ const FieldNoLabel = ({ field, form, widgetElem, children }) => {
       {getHelpBlock(field, form)}
       {children}
       <FormControl.Feedback
-        type={validationState ?? "invalid"}
+        type={validationState && "invalid"}
```

Comment:
Why not have `getFormGroupValidationState` return `"invalid"` on validation errors?